inst/extdata/packages/rgbif.yaml

```diff
@@ -0,0 +1,39 @@
+name: rgbif
+title: Interface to the Global Biodiversity Information Facility API
+version: 3.7.7
+author: Scott Chamberlain, Damiano Oldoni, Vijay Barve, Peter Desmet, Laurens Geffert,
+  Dan Mcglinn, Karthik Ram, rOpenSci, John Waller
+maintainer: John Waller <jwaller@gbif.org>
+cran: yes
+repository: https://github.com/ropensci/rgbif
+description: A programmatic interface to the Web Service methods provided by the Global
+  Biodiversity Information Facility (GBIF; <https://www.gbif.org/developer/summary>).
+  GBIF is a database of species occurrence records from sources all over the globe.
+  rgbif includes functions for searching for taxonomic names, retrieving information
+  on data providers, getting species occurrence records, getting counts of occurrence
+  records, and using the GBIF tile map service to make rasters summarizing huge amounts
+  of data.
+occ_acquisition: yes
+occ_cleaning: no
```

Comment:
It actually does this now: http://cran.nexr.com/web/packages/rgbif/vignettes/issues_vignette.html